app/controllers/manual_sections_controller.rb

```diff
@@ -12,6 +12,30 @@ def show
     Airbrake.notify(e)
   end
 
+  def new
+    @section = Section.new(manual_content_id: params["manual_content_id"])
+  end
+
+  def create
+    section_data = params["section"]
+    section_data["manual_content_id"] = params["manual_content_id"]
+
+    @section = Section.new(section_data)
+
+    if @section.valid?
+      if @section.save!
```

Comment:
`save!` will raise if it fails so the `if\else` won't happen. `save` returns a boolean. It also checks `valid?` so you could shorten this down to `if save` and remove the outer `valid?` branch. If an error is raised along the way then it will fall through to the default error page which I would say is as informative as the error message you're trying to return. It will also presumably get reported back to Errbit.